packages/test-apps/react-vite/src/studio-config.ts

```diff
@@ -6,181 +6,188 @@ import NestedSlots from './components/NestedSlots';
-        },
-        date: {
-          displayName: 'Date',
-          type: 'Date',
-        },
-        location: {
-          displayName: 'Location',
-          type: 'Location',
-          defaultValue: { lon: -69, lat: 69 },
-        },
-        media: {
-          displayName: 'Media',
-          type: 'Media',
-          defaultValue:
-            'https://images.ctfassets.net/son9ld5ewssk/66dcJbD6GoCqwQ0ciGaCOg/6c20042a8684baedc1517fadfc3e3070/children-enabled_7x.png',
-        },
-        hyperlink: {
-          displayName: 'Hyperlink',
-          type: 'Hyperlink',
-          defaultValue: 'https://www.contentful.com',
-        },
-        richText: {
-          displayName: 'Rich Text',
-          type: 'RichText',
+    {
+      component: KitchenSink,
+      definition: {
+        id: 'kitchen-sink',
+        thumbnailUrl:
+          'https://images.ctfassets.net/son9ld5ewssk/66dcJbD6GoCqwQ0ciGaCOg/6c20042a8684baedc1517fadfc3e3070/children-enabled_7x.png',
+        children: true,
+        name: 'Kitchen Sink',
+        category: 'Custom Components',
+        builtInStyles: ['cfMargin', 'cfPadding', 'cfWidth', 'cfMaxWidth'],
+        variables: {
+          jsonObject: {
+            displayName: 'Json',
+            type: 'Object',
+          },
+          booleanValue: {
+            displayName: 'Boolean',
+            type: 'Boolean',
+          },
+          listText: {
+            displayName: 'List Text',
+            type: 'Array',
+          },
+          entryReference: {
+            displayName: 'Entry Reference',
+            type: 'Link',
+          },
+          imageReference: {
+            displayName: 'Image Reference',
+            type: 'Link',
+          },
+          listReference: {
+            displayName: 'List Entry Reference',
+            type: 'Array',
+          },
+          text: {
+            displayName: 'Text',
+            type: 'Text',
+            // defaultValue: 'Lorem ipsum',
+          },
+          date: {
+            displayName: 'Date',
+            type: 'Date',
+          },
+          location: {
+            displayName: 'Location',
+            type: 'Location',
+            defaultValue: { lon: -69, lat: 69 },
+          },
+          media: {
+            displayName: 'Media',
+            type: 'Media',
+            defaultValue:
+              'https://images.ctfassets.net/son9ld5ewssk/66dcJbD6GoCqwQ0ciGaCOg/6c20042a8684baedc1517fadfc3e3070/children-enabled_7x.png',
+          },
+          hyperlink: {
+            displayName: 'Hyperlink',
+            type: 'Hyperlink',
+            defaultValue: 'https://www.contentful.com',
+          },
+          richText: {
+            displayName: 'Rich Text',
+            type: 'RichText',
+          },
         },
       },
+      options: {
+        wrapComponent: false,
+      },
     },
-    options: {
-      wrapComponent: false,
+  ],
+  {
+    experimentalComponents: {
+      carousel: true,
     },
   },
```

Comment:
Could you also update the `ComponentDefinitionOptions` type to include this new option?

[packages/core/src/types.ts#L120-L122](https://github.com/contentful/experience-builder/blob/development/packages/core/src/types.ts#L120-L122)

Then TS will know about this new option when it's implemented in the `defineComponents` function.

Reply:
yeah i tried to before but git didnt seem to pick up on the change 🤔